hgtv/forms/playlist.py

```diff
@@ -24,7 +24,8 @@ class PlaylistForm(Form):
         description=u"The name of your playlist")
     name = wtforms.TextField(u"URL Name", validators=[wtforms.validators.Optional()],
         description=u"Optional. Will be automatically generated if left blank")
-    description = RichTextField(u"Description")
+    description = TinyMce4Field(u"Description",
+        tinymce_options={'height': 200, 'plugins': 'autoresize', 'autoresize_min_height': 150, 'autoresize_max_height': 200})
```

Comment:
This is UI detail. Should it be here in the form? Should we make these the default options for TinyMCE if they are what we typically use? (Asking since we have this autoresize feature in Hasjob as well, the only other place we use TinyMCE.)

Reply:
Fixed. Set these as default for TinyMCE in baseframe.